src/ontology/cl-edit.owl

```diff
@@ -28265,6 +28257,82 @@ AnnotationAssertion(rdfs:label obo:CL_4023081 "inverted L6 intratelencephalic pr
 SubClassOf(obo:CL_4023081 obo:CL_4023050)
 SubClassOf(obo:CL_4023081 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070021))
 
+# Class: obo:CL_4023083 (chandelier cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15378039") obo:IAO_0000115 obo:CL_4023083 "A type of interneuron that are axon-targeting. They can be multipolar or bitufted. Their local axonal clusters are formed by high-frequency branching at shallow angles, often ramifying around, above or below their somata with a high bouton density. The characteristic terminal portions of the axon form short vertical rows of boutons, resembling a chandelier.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023083 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(obo:IAO_0000116 "Interlex cross reference refers specifically to Markram 2015 rat chandelier cell.") oboInOwl:hasDbXref obo:CL_4023083 "ILX:0777213")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15378039") oboInOwl:hasExactSynonym obo:CL_4023083 "ChC")
+AnnotationAssertion(rdfs:label obo:CL_4023083 "chandelier cell")
+EquivalentClasses(obo:CL_4023083 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070011)))
+
+# Class: obo:CL_4023086 ('T-shaped' Martinotti neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:10.1016/b978-0-12-369497-3.10004-4"^^xsd:string) obo:IAO_0000115 obo:CL_4023086 "A Martinotti neuron that has axons that form a horizontal ramification, making it T-shaped.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_4023086 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:inSubset obo:CL_4023086 <http://purl.obolibrary.org/obo/cl#BDS_subset>)
+AnnotationAssertion(rdfs:label obo:CL_4023086 "'T-shaped' Martinotti neuron")
+EquivalentClasses(obo:CL_4023086 ObjectIntersectionOf(obo:CL_4023076 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070008)))
+
+# Class: obo:CL_4023087 ('fanning-out' Martinotti neuron)
```

Comment:
This is really nit-picky, but why does the label have shudder quotes? The hyphenated gerund+preposition is a bit weird to me as well. I suppose that is historical precedent, though.

Reply:
yeah it is a PAIN to add the PATO term as an axiom - protege really does not like have two ' consecutively and I have been doing a work around by removing it, then adding it back in after I have placed the axiom. I really would like the remove it, but at the moment I'm directly copying it from the paper. Thoughts on this @dosumis?

Reply:
Ah, no problem then.

Reply:
Please lose the shudder quotes.  If you want to keep them for search purposes, add a synonym.

Reply:
Maybe 'fan Martinotti neuron' for the label - if you can find precedent.   Not a big deal though.

Reply:
will remove them, and will update PATO accordingly too. Thanks